client/src/components/Chat/MessagesHeader/MessagesHeader.jsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styles from './MessagesHeader.module.scss';
+
+function MessagesHeader() {
+  const current = useSelector((state) => state.chat.currentPerson);
```

Comment:
current user data can also be taken from the parent component through props instead of duplicating call to store